core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -877,6 +877,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey HOST_SSH_PORT =
+          intBuilder(Name.HOST_SSH_PORT)
```

Comment:
Please keep the indent consistent with the other propertykey defination.

Reply:
Done